internal/testrunner/runners/asset/runner.go

```diff
@@ -0,0 +1,167 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package asset
+
+import (
+	"fmt"
+	"path/filepath"
+	"time"
+
+	es "github.com/elastic/go-elasticsearch/v7"
+	"github.com/pkg/errors"
+
+	"github.com/elastic/elastic-package/internal/kibana/ingestmanager"
+	"github.com/elastic/elastic-package/internal/logger"
+	"github.com/elastic/elastic-package/internal/packages"
+	"github.com/elastic/elastic-package/internal/testrunner"
+)
+
+func init() {
+	testrunner.RegisterRunner(runner{})
+}
+
+const (
+	// TestType defining asset loading tests
+	TestType testrunner.TestType = "asset"
+)
+
+type runner struct {
+	testFolder      testrunner.TestFolder
+	packageRootPath string
+	stackSettings   testrunner.StackSettings
+	esClient        *es.Client
+
+	// Execution order of following handlers is defined in runner.tearDown() method.
+	removePackageHandler func()
+}
+
+// Type returns the type of test that can be run by this test runner.
+func (r runner) Type() testrunner.TestType {
+	return TestType
+}
+
+// String returns the name of the test runner.
+func (r runner) String() string {
+	return "asset loading"
+}
+
+// CanRunPerDataStream returns whether this test runner can run on individual
+// data streams within the package.
+func (r runner) CanRunPerDataStream() bool {
```

Comment:
Let's keep it consistent with README: https://github.com/elastic/elastic-package#elastic-package-test . A context can be global, package or data stream.

Reply:
I'm not sure I follow what to do here. Would you mind elaborating?

In my mind, `elastic-package test` is a command with `package` context. It just allows for extra granularity, if the user desires, via the `--data-stream` flag.

Reply:
I'm sorry but I can't remind myself what I was thinking about... Let's just ignore this comment.